components/date-picker/interface.tsx

```diff
@@ -31,6 +31,7 @@ export interface SinglePickerProps {
   defaultValue?: moment.Moment;
   defaultPickerValue?: moment.Moment;
   placeholder?: string;
+  renderExtraFooter?: (mode: 'date' | 'month' | 'year' | 'decade') => React.ReactNode;
```

Comment:
use `DatePickerMode` instead.

Reply:
@zombieJ done.

I also added `"decade"` to `DatePickerMode`, that was missing there. See commit for link to playground.